### ISSUE TITLE
How to localize or translate properly

### ISSUE THREAD

**Issue by kiraware** [angryip/ipscan#451]
Hello everyone, I want to help the localization of angry ip for Bahasa Indonesia (id). I have read translation guide at https://angryip.org/contribute/. I know I have to translate `messages.properties` to `messages_id.properties`. The problem comes when i read that content of the files. There is good text too translate and it's clear how to translate it, but sometimes there is a text that contain the ampersand character (&) in the middle of the text. I don't know what that means.

For example on this line https://github.com/angryip/ipscan/blob/16bb80bd16e765020d1c6bb5cb7eaecf8d91d940/resources/messages.properties#L67
As you can see, it looks weird, and i have no idea how to translate that text.

Also in these lines
https://github.com/angryip/ipscan/blob/16bb80bd16e765020d1c6bb5cb7eaecf8d91d940/resources/messages.properties#L11-L26
Am I responsible to add `language.id=Bahasa Indonesia`? Since there is no Indonesian translation before. I hope someone could give me guide how to translate it properly. Tagging maintainer @angryziber, if someone could point me out, I'll soon add a PR for Bahasa Indonesia. Thank you

**Comment by JonnyTech** [comment]
The ampersand symbol in GUI applications is usually indicative that the following character is a keyboard shortcut, therefore from your above example, `Without o&pen ports` means that the menu item is selected by <kbd>ALT</kbd>+<kbd>P</kbd> - so look to see how other translators have done it.

**Comment by kiraware** [comment]
> The ampersand symbol in GUI applications is usually indicative that the following character is a keyboard shortcu

To make it clear, for example

`Without o&pen ports` is ALT+P
`Without op&en ports` is ALT+E
`Without ope&n ports` is ALT+N

Am I right?

Also how about the new language, should I added `language.id=Bahasa Indonesia` to `messages.properties` as I mentioned above?

**Comment by JonnyTech** [comment]
For the ampersand query, yes your examples are correct. Look at the other translations to see if they either selected another shortcut or removed it.

For the new language addition, yours is a reasonable assumption, but best wait for @angryziber to confirm.

**Comment by kiraware** [comment]
Thank you for your response

**Comment by angryziber** [comment]
@kiraware yep, please create messages_id.properties and either submit a Pull Request or attach it here
